docs/integrations/databases/opentelemetry/postgresql-opentelemetry.md

```diff
@@ -0,0 +1,278 @@
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PostgreSQL-OpenTelemetry/PostgreSQL-Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for PostgreSQL
+
+Following are the tags which will be created as part of PostgreSQL App install if not already present: 
+
+- **db.cluster.name** - User configured. Enter a name to identify this Postgresql cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **db.system** - Has a fixed value of **postgresql**.
+- **sumo.datasource** - Has a fixed value of **postgresql**.
+- **db.node.name** - Has a value of the host name of the machine which is being monitored.
+
+## Prerequisites
+
+[Here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/postgresqlreceiver#prerequisites) are the prerequisites for metric collection.
+
+Configure logging in PostgreSQL
+
+1.  Locate your local PostgreSQL postgresql.conf configuration file in the database data_directory. For more information, see the [PostgreSQL File Locations documentation](https://www.postgresql.org/docs/9.1/static/runtime-config-file-locations.html). By default it's located in `/var/lib/pgsql/<version>/data/postgresql.conf`. You can run SHOW config_file command inside your server's psql shell to get the location. After determining the location of conf file, modify the PostgreSQL postgresql.conf configuration file logging parameters
+2.  Connect to the database server (using SSH) in a terminal window.
+3.  Open `postgresql.conf` configuration file.
+4.  Under the ERROR REPORTING AND LOGGING section of the file, use the following config parameters. For more information on the following parameters, [click here](https://www.postgresql.org/docs/12/static/runtime-config-logging.html).
+```sql
+    log_destination = 'stderr'
+    logging_collector = on
+    log_filename = 'postgresql-%Y-%m-%d_%H%M%S.log'
+    log_truncate_on_rotation = off
+    log_rotation_age = 1d
+    log_min_duration_statement = 250
+    log_connections = on
+    log_duration = on
+    log_hostname = on
+    log_timezone = 'UTC'
+    log_min_messages = 'WARNING'
+    log_line_prefix = '%m [%p] %q%u@%d '
+```
+5.  Save the `postgresql.conf` file and restart the postgresql server: `sudo  service postgresql restart`
+
+## Installing the App for PostgreSQL
+
+If you want to create a new collector please select the **Add a new collector** option.
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
```

Comment:
```suggestion
To create a new Collector:
	
1. Select the **Add a new Collector** option.
2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.
	
This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.

```